tests/test_sum/test_another_sum.py

```diff
@@ -1,9 +1,12 @@
 import unittest
-from sum.another_sum import another_sum
+from sum.another_sum import another_sum, adD10
 
 class MyTestCase(unittest.TestCase):
     def test_another_sum(self):
         assert another_sum(3, 2) == 5
 
+    def test_one10(self):
+        assert adD10(3) == 13
```

Comment:
Same problem of function naming. Choose add_10

Reply:
Good comment!